docs/tasks/active/20260505-undo-destroys-initial-block-todo.md

```
@@ -0,0 +1,79 @@
+# Undo destroys initial block
+
+**Created**: 2026-05-05
+
+## Problem
+
+Reproduction: open a freshly-created docs document, type "asdf",
+press Enter, type "asdf", press Enter, type "asdf", then press
+Cmd+Z enough times. The next keystroke crashes with:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)